### PR TITLE
added run scripts + fixed pom.xml to make jars with dependencies + fix skija perf

### PR DESCRIPTION
**Pull request by bourgesl**
Some changes to run tests easily

### CODE REVIEW

File: g2d-core/run.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source ~/test-jdk17.sh
```
Comment (jfree) [inline g2d-core/run.sh]: What does this one do?

File: g2d-fxgraphics2d/run.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+source ~/test-zulu17.sh
```
Comment (jfree) [inline g2d-fxgraphics2d/run.sh]: What does this one do?

**Comment by bourgesl** [comment]
You're right. Good catch.

This scripts do set appropriate jdk: jdk17 and zulu17 (jdk + fx) to work.

I will see how to generalize it ...

**Comment by bourgesl** [comment]
Probably I will add these scripts to easily set appropriate jdk paths to run all tests with a new run-all.sh script ...
Stay tuned.
FYI fx g2d seems to not handle properly alpha composite tests (porter-duff rules)... 
I will share my png files and report or fix any bug found...
😉

File: g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java [hunk 3]
```diff
@@ -2,36 +2,62 @@
...
+        final SkijaGraphics2D g2 = new SkijaGraphics2D(width, height);
         try {
-            if (single) {
-                fileName += "-single.png";
-            } else {
-                fileName += ".png";
+            for (int i = 0; i < REPEATS; i++) {
+                final long startTime = System.nanoTime();
+
+                Tester.drawTestOutput(g2, "SkijaGraphics2D 1.0.4", "https://github.com/jfree/skijagraphics2d", single);
```
Comment (jfree) [inline g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java:30]: Should update the version here - it appears in the sample output.

Comment (bourgesl) [reply g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java:30]: I fixed it in my local copy...

**Review by jfree — APPROVED** [review]

File: g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java (hunk 3 above)
Comment (jfree) [reply g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java:30]: You mentioned that you'll do a follow up PR, so just include it in that.  I'll merge this one.

**Pull request merged**